packages/amplify-category-function/src/commands/function/remove.ts

```diff
@@ -29,8 +31,10 @@ module.exports = {
         }
       })
       .catch(err => {
-        context.print.info(err.stack);
-        context.print.error('An error occurred when removing the function resource');
+        if (err.stack) {
+          context.print.info(err.stack);
+          context.print.error('An error occurred when removing the function resource');
+        }
```

Comment:
Should we print the stack trace here?

Reply:
If it is an unhandled error then sure, helps with troubleshooting, but this kind of handling was added in the main package as well to make sure user friendly error messages will not be followed by technical data like stack trace.